Assets/Scripts/PlayerInteractionController.cs

```diff
@@ -77,10 +77,13 @@ private void Update()
                     animator.SetBool("action", true);
                     Debug.unityLogger.Log(animator.GetBool("action"));
                 }
-                else if (_itemInMind != closestItem)
+                else
                 {
-                    _itemInMind = closestItem;
-                    SetThoughtForInteractableItem(closestItem);
+                    if (_itemInMind != closestItem)
+                    {
+                        _itemInMind = closestItem;
+                    }
+                    SetThoughtForInteractableItem(_itemInMind);

```

Comment:
Thought about this thing again. When we set the thought, we always re-create the game object, which may be a tad expensive. Maybe we should make item bubble remember the sprites it rendered, so it could avoid remaking the same object? May be done as a part of my PR (#44).

Reply:
That needs further discussion and design. Good point but I don't think it's going to cost us too much and I'd rather not start refactoring every small thing that isn't a big issue, I think we should focus on making more progress on new things.